general.py

```diff
@@ -0,0 +1,102 @@
+#### General
+#1.  Create a function that accepts an array of integers and returns the sum of all the even numbers in the array.
+def sumOfEven(array):
+    return(sum([
+        item if item%2==0 else 0 for item in array
+    ]))
+
+def sumOfEvenTest():
+    print(sumOfEven([0,1,2,3]))#2
+    print(sumOfEven([1,3,5]))#0
+    print(sumOfEven([0,6,2,7]))#8
+
+#sumOfEvenTest()
+
+#2. Create a function that accepts two arrays and returns a new array that contains only the elements that are common between the two arrays.
+def intersection(firstArray,secondArray):
+    firstArray = set(firstArray)
+    secondArray = set(secondArray)
+    result = firstArray.intersection(secondArray)
+    result = list(result)
+    return(result)
+
+def intersectionTest():
+    print(intersection([1,2,3],[4,5,6]))#[]
+    print(intersection([1,2,3],[1,2,3]))#[1,2,3]
+    print(intersection([1,2,3],[1,3,4]))#[1,3]
+    print(intersection([1,2,3],[6,2,2]))#[2]
+
+#intersectionTest()
+
+#3. Create a function that accepts an array of integers and returns the second smallest number in the array.
```

Comment:
You can do this with list.sort and you can use slicing (list[1]) to get the second smallest

Reply:
you will also need to make it a set before hand